### PR TITLE
Add warning to domainId maximum

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/744/)

File: docs/fastdds/dds_layer/domain/domainParticipant/createDomainParticipant.rst [hunk 1]
```diff
@@ -27,6 +27,14 @@ Optional arguments are:
    :ref:`dds_layer_domainParticipantListener`.
    By default all events are enabled.
 
+.. _DDSI-RTPS V2.2: https://www.omg.org/spec/DDSI-RTPS/2.2/PDF
+
+.. warning::
+   Following the `DDSI-RTPS V2.2`_ standard (Section 9.6.1.1), the default ports are calculated depending on the
+   domainId, as it is explained in section :ref:`listening_locators_defaultPorts`.
+   Thus, it is encouraged to use domainIds lower than 200
```
Comment (EduPonz) [inline docs/fastdds/dds_layer/domain/domainParticipant/createDomainParticipant.rst:35]: Either use an alias or a literal for `domainId`.

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Review by EduPonz — APPROVED** [review]
LGTM with green CI

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/780/)

**Pull request merged**